Exercise_4/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+namespace Exercise_4
+{
+    class Program
+    {
+        static List<string> FilterFile(string root, Predicate<string> predicate)
+        {
+            var files = new List<string>();
+            
+            foreach (var file in Directory.GetFiles(root))
```

Comment:
Get all files in a folder and all sub folders

Reply:
Done